### PR TITLE
#160609546 Setup and deploy base application to Heroku

### PR DESCRIPTION
**Pull request by RonKbS**
### What does this PR do?

Configures the application so that builds on Heroku are successful.

#### Description of Task to be completed.
After successful merging into the develop and master branches, by accessing the URLs below for the application in staging and production environments, the application can be accessed remotely.

#### How should this be manually tested?
https://ah-backend-targaryen-staging.herokuapp.com/
https://ah-backend-targaryen.herokuapp.com/

#### What are the relevant pivotal tracker stories?
[#160609546](https://www.pivotaltracker.com/story/show/160609546)

### CODE REVIEW

**Comment by kakaemma** [comment]
Kindly make the story ID link to the respective story on pivotal tracker.

**Review by samuelbwambale — APPROVED** [review]
@RonKbS the PR looks fine

**Review by mendozabree — APPROVED** [review]
LGTM

**Comment by isheebo** [comment]
Please fix the merge conflicts and let me know

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+release: python manage.py migrate
+web: gunicorn authors.wsgi --log-file -
```
Comment (isheebo) [inline Procfile]: add new line here

File: app.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "DATABASE_URL": {
+        "required": true
+      }
+    },
+    "formation": {},
+    "addons": [
+      "heroku-postgresql"
+    ],
+    "buildpacks": []
+  }
```
Comment (isheebo) [inline app.json]: add new line here

**Comment by isheebo** [comment]
Please re-edit your commit message. Reduce the number of things included in the body of the commit message

**Comment by RonKbS** [comment]
@isheebo the merge conflicts have been resolved, commit message has been edited and new lines have been added in the respective files.

**Pull request merged**